Interface/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ini8labs/Go-Learning/Interface/carPrice"
+)
+
+func totalExpense(s []carPrice.Price) float64 {
+
+	Total := 0.0
+	for _, v := range s {
+		Total += v.PriceCalculator()
+	}
+	return Total
+}
+
+func main() {
+	car1 := carPrice.Beemer{
+		Name:      "BMW",
+		Yom:       2020,
+		BasePrice: 300000}
+	car2 := carPrice.Toyota{
```

Comment:
space after defining car1, car2, car3
Instead of car1, car2, car3, car4 give names beemer1, beemer2, toyota1, toyota2